pkg/controller/migplan/migplan_controller.go

```diff
@@ -43,6 +43,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+const (
+	MigPlan   = "migplan"
+	CreatedBy = "CreatedBy"
+)
+
+// define maximum waiting time for mig analytic to be ready
+var timeLimit = 3 * time.Minute
```

Comment:
please make this variable more specific: `migAnalyticsTimeout`

Reply:
Done